bower.json

```diff
@@ -62,5 +62,8 @@
         "build/jsfeat.js"
       ]
     }
+  },
+  "resolutions": {
```

Comment:
They're were problems with the installation of jsfeat. I couldn't figure
out why so I just ignored it. Does it tell you what the error is? I could
try reinstalling and send you the error message

On Wed, Jan 13, 2016 at 2:52 PM, Alexandr Kalinin <notifications@github.com>
wrote:

> In bower.json
> <https://github.com/SOCR/SOCRAT/pull/52#discussion_r49664015>:
>
> > @@ -62,5 +62,8 @@
> >          "build/jsfeat.js"
> >        ]
> >      }
> > +  },
> > +  "resolutions": {
>
> Why is this in <resolutions> section and not in <dependencies>? Were
> there problems during installation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SOCR/SOCRAT/pull/52/files#r49664015>.
>



-- 
Nellie Ponarul
B.S. in Applied Mathematics
University of California, Los Angeles | 2017
nellie.ponarul@ucla.edu